lib/google-analytics.js

```diff
@@ -52,30 +44,35 @@ module.exports = (function () {
 
     GoogleAnalytics.getCid(function (cid) {
       _.extend(params, { cid: cid }, GoogleAnalytics.defaultParams());
-      GoogleAnalytics.request('https://www.google-analytics.com/collect?' + querystring.stringify(params));
+      this.request('https://www.google-analytics.com/collect?' + querystring.stringify(params));
```

Comment:
I'll end up testing this in my own environment when I'm done, so any glaring issues should be taken care of then.